From 504f6872cd5205d35b772ca4796199c75aad233b Mon Sep 17 00:00:00 2001
From: Wu Zhangjin <zhangjin.wu@windriver.com>
Date: Sun, 26 Sep 2010 07:29:50 -0700
Subject: [PATCH 2/2] tracing/trace-clock: omap2: Add a missing raw_local_irq_save conversion

"tracing: don't trace trace-clock" has converted local_irq_* to the raw
ones, but has missed the one in hotcpu_callback(), this patch adds it.

Signed-off-by: Wu Zhangjin <zhangjin.wu@windriver.com>
---
 arch/arm/mach-omap2/trace-clock.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/trace-clock.c b/arch/arm/mach-omap2/trace-clock.c
index d0bfd76..bf50dcc 100644
--- a/arch/arm/mach-omap2/trace-clock.c
+++ b/arch/arm/mach-omap2/trace-clock.c
@@ -493,7 +493,7 @@ static int __cpuinit hotcpu_callback(struct notifier_block *nb,
 		spin_lock(&trace_clock_lock);
 		if (trace_clock_refcount) {
 			pm_count = &per_cpu(pm_save_count, hotcpu);
-			local_irq_save(flags);
+			raw_local_irq_save(flags);
 			pm_count->ext_32k = clock->read(clock);
 			pm_count->int_fast_clock = trace_clock_read64();
 			raw_local_irq_restore(flags);
-- 
1.6.5.2

